vite.config.ts

```diff
@@ -2,17 +2,26 @@ import { defineConfig } from '@ownclouders/extension-sdk'
 import { readFileSync } from 'fs'
 import { join } from 'path'
 
-const certsDir = `${__dirname}/dev/docker/traefik/certificates`
+const isProduction = process.env.PRODUCTION
```

Comment:
env value will always be a string
```suggestion
const isProduction = process.env.PRODUCTION === 'true'
```